### PR TITLE
Support pasting username with autofill, fixes #192

### LINKED ISSUE #192: Add more fields to autofill
I think it will be very useful to have easy bash to input login automatically. You can see example in pass firefox extension. It gets login name from 
"login: lapels" string, etc.

### PR DESCRIPTION
**Pull request by felixb**
The workflow for pasting usernames is as follows:

1. Select password field
2. Select password store entry with username and paste it
3. Select any other editable field
4. Paste username

### CODE REVIEW

File: app/src/main/java/com/zeapo/pwdstore/autofill/AutofillService.java [hunk 1]
```diff
@@ -58,16 +59,21 @@
...
     private String webViewTitle = null;
     private String webViewURL = null;
+    private PasswordEntry lastPassword;
+    private long lastPasswordDate;
 
-    public final class Constants {
-        public static final String TAG = "Keychain";
+    final class Constants {
+        static final String TAG = "Keychain";
+        static final long MAX_PASSWORD_ENTRY_LIFETIME = 45 * 1000;
```
Comment (zeapo) [inline app/src/main/java/com/zeapo/pwdstore/autofill/AutofillService.java]: We could use the same timeout value as the copy/paste

**Comment by zeapo** [comment]
Thanks for the PR.

In the workflow, could you add a notification (snack or toast, chose what you think is better) to inform the user that the selected password has a username in it and that the password was put in cache for `x` seconds so that he selects the username field to paste it.

File: app/src/main/java/com/zeapo/pwdstore/autofill/AutofillService.java (hunk 1 above)
Comment (zeapo) [inline app/src/main/java/com/zeapo/pwdstore/autofill/AutofillService.java]: It's `general_show_time`, we should rename it as it's used for visibility and clipboard timeouts.

**Comment by felixb** [comment]
I added the toast in a 2nd commit for easier review. Feel free to squash the commits when merging.

**Comment by zeapo** [comment]
Works perfectly, thanks.

**Pull request merged**